src/Cache/Engine/RedisEngine.php

```diff
@@ -101,6 +115,51 @@ public function init(array $config = []): bool
      * @return bool True if Redis server was connected
      */
     protected function _connect(): bool
+    {
+        if (!empty($this->_config['nodes'])) {
+            return $this->_connectRedisCluster();
+        }
+
+        return $this->_connectRedis();
+    }
+
+    /**
+     * Connects to a Redis cluster server
+     *
+     * @return bool True if Redis server was connected
+     */
+    protected function _connectRedisCluster(): bool
```

Comment:
```suggestion
    protected function connectRedisCluster(): bool
```

We don't use underscore prefix for new protected methods anymore.

Reply:
Good old "half pregnant" approach where half of the code base has underscore and other half is updated

Reply:
Half isn't updated, we just stopped adding underscores at one point. We hope to address this "half pregnancy" in the next major :slightly_smiling_face: https://github.com/cakephp/cakephp/issues/17862